src/main/java/core/Main.java

```diff
@@ -0,0 +1,53 @@
+package core;
```

Comment:
```suggestion
package core.basesyntax;
```
and every other package should be in `core.basesyntax`